libtool/.gitignore

```diff
@@ -2,3 +2,7 @@ bin/
 include/
 lib/
 share/
+src/gmp-*
+/CMakeFiles
+/Makefile
+/cmake_install.cmake
```

Comment:
Is it correct to add a slash `/` at the beginning? Will it affect the Makefile at the root?

Reply:
The leading slash means that this only apply to current directoy. `Makefile` in project root was already in repository and I want to ignore these files just in libtool directory.

Reply:
Every place that I search says that a leading slash means the root folder, so it is better to remove the `/` from the beginning

Reply:
OK, I was reading wrong docs